handlers/EveryoneAndHereMentionMessage.js

```diff
@@ -0,0 +1,24 @@
+const Discord = require('discord.js');
+
+const color = '#eb1540';
+const title = 'Nie wołaj wszystkich!';
+const thumbnail = 'LINK'
+const description = '\nRozumiemy, że potrzebujesz pomocy, ale nie wszyscy chcą zostać o tym powiadomieni.\n Jest nas tutaj dużo i nie ma sensu, aby każdy dostał bezpośrednio taką informację.\n Nie trudno sobie wyobrazić jak irytujące byłoby, gdyby każdy wołał wszystkich do każdego tematu.\n Dlatego zadaj pytanie i po prostu poczekaj - jeśli ktoś będzie wiedział i mógł, to na pewno spróbuje odpowiedzieć.'
+
+export function mentionHandler(){
+    const pingEmbed = new Discord.MessageEmbed()
+	.setColor(color)
+	.setTitle(title)
+    .setThumbnail(thumbnail)
+    .setDescription(description);
+
+    const mentionedEveryone = message.content.includes("@everyone") || message.content.includes("@here")
```

Comment:
Te stringi `@everyone` i `@here` też można wrzucić w zmienne, albo nawet lepiej, zgrupować w jakiś słownik.

Reply:
Skąd ta funkcja ma dostęp do message? Nie widzę tu podawania referencji w argumentach. Rozumiem że sprawdziłeś to i działa?

Reply:
> Te stringi `@everyone` i `@here` też można wrzucić w zmienne, albo nawet lepiej, zgrupować w jakiś słownik.

Myślę, że nie ma takiej potrzeby, gdyż i tak raczej nowe pingi się nie pojawią.

Reply:
> Myślę, że nie ma takiej potrzeby, gdyż i tak raczej nowe pingi się nie pojawią.

Chodzi o to, że te stringi kontekstowo należą do pewnej grupy - odpowiadają za pingowanie userów. Wyobraź sobie sytuację, że funkcjonalność jest rozszerzana i w kilku miejscach korzystasz z tych samych stringów + dochodzą jakieś inne stringi odpowiadające np. komendom (niech będzie `/test`). W pewnym momencie trudno jest Ci się połapać w kodzie co kontekstowo oznacza dany string, a jakbyś chciał jednak zmienić go w kilku miejscach to musisz te kilka miejsc edytować. W momencie, gdy wrzucisz takie stringi to słownika, to możesz je sobie zgrupować i wtedy szybciej i wygodniej jest się tym posługiwać, np. `DICTIONARY.PINGS.EVERYONE` i `DICTIONARY.PINGS.HERE` są łatwiejsze w interpretacji przez programistę niż jakiś "randomowy" string '@everyone' w kodzie. Poza tym, jeśli np. chcesz sprawdzić w apce jakie podobne stringi są używane, to szukasz pierwszego lepszego (albo idziesz bezpośrednio do miejsca deklaracji słownika) i odnosisz się do jego obiektu, gdzie masz spis wszystkich kontekstowo podobnych stringów - w jednym miejscu (które można na dodatek sobie udokumentować).